### PR TITLE
Feat(Settings): Email Panel Styles

### PR DESCRIPTION
**Pull request by Chrissy**
emails

todo:

- [x] storybook for return button
- [x] snapshot tests for return button
- [x] don't use setState(?)

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Review by billyvg — COMMENTED** [review]
Looks good!

I'm not sure if the click functionality is necessary, the fields where we want to show it, will save on blur or with an enter keypress. But I suppose users will want to click it, and it's better to have it do what users will expect (save).

I don't see anything with state management?

**Comment by Chrissy** [comment]
@billyvg yah sorry "state management" is ambiguous. I just meant that I'm using basic react `setState`, while everything else in the component seems to be operating through `FormState`.

But sometimes the simple thing is the best thing! So if you think it seems fine let's do it. I _do_ need to change state tho because we gotta trigger a repaint on field change (so I can show the button).

File: src/sentry/static/sentry/app/data/forms/accountEmails.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const forms = [
         label: 'Additional Email',
         placeholder: 'e.g. secondary@example.com',
         help: 'Designate an alternative email for this account',
+        submitOnReturn: true,
```
[the remark this answers is not in the document]
Comment (Chrissy) [reply src/sentry/static/sentry/app/data/forms/accountEmails.jsx]: sorry this is a dumb name. It just shows the button. I'm open to ideas!

**Comment by billyvg** [comment]
I don’t see any setstate calls in the diffs, did you forget to commit?

**Comment by Chrissy** [comment]
@billyvg ugh yah sorry 😴 

It's pushed now.

File: src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx [hunk 2]
```diff
@@ -158,7 +171,7 @@ class FormField extends React.Component {
 
   // Only works for styled inputs
   handleInputMount = ref => {
-    if (ref && !this.input) {
+    if (ref && !this.input && this.context.location) {
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx]: This is for storybook compatibility and might not be the right solution

File: src/sentry/static/sentry/app/views/settings/components/tag.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  color: #fff;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  border-radius: 0.25em;
+  margin-left: 0.5em;
+  text-transform: lowercase;
+  background-color: ${p => {
+    switch (p.type) {
+      case 'warning':
```
[the remark this answers is not in the document]
Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/components/tag.jsx]: Looking at this again, the colors are already in `theme.js`. do you mean adding secondary names like `warning` and `success`?

[the remark this answers is not in the document]
Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/components/tag.jsx]: nvm disregard we already have this stuff started in `theme.js`

**Review by billyvg — APPROVED** [review]
This looks good, we'll have to do some refactoring to make it behave well with "save on blur" fields, but we can worry about that later

**Pull request merged**